### PR TITLE
Trends JSX Content and Css Outline

### PR DESCRIPTION
**Pull request by Joe-Bentley**

### CODE REVIEW

File: src/components/Trends.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import Biden from '../images/trends-biden.jpg';
+import TrendsFood from '../images/trends-food.jpg';
+
+
+const Trends = () => {
+
+    return(
+
+    <div className="trends-panel">
+        {/* <div className="searchBar" > */}
```
Comment (nmunro) [inline src/components/Trends.js:13]: Is this a placeholder for something that will be added later?

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "@testing-library/user-event": "^12.1.10",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^3.11.0",
```
Comment (tamjankowska) [inline package.json:12]: React icons have been installed here as a dependency.

File: src/components/Trends.js (hunk 1 above)
Comment (Joe-Bentley) [reply src/components/Trends.js:13]: Its a CSS className / div className Issue which will be fixed

**Review by tamjankowska — APPROVED** [review]

File: src/components/Trends.js (hunk 1 above)
Comment (nmunro) [reply src/components/Trends.js:13]: Ah ok, cool :)

**Pull request merged**